docs/modules/help/partials/faq_ticket_edit.adoc

```diff
@@ -0,0 +1,38 @@
+[[creation]]
+=== When is VSHN notified when I create a new ticket?
+
+VSHNeers are notified immediately of the creation of new tickets. Higher priority tickets trigger additional internal alert systems for a faster response.
+
+.Example e-mail to VSHNeers after ticket activity
+image::ticket-email.png[]
+
+[[events]]
+=== What happens after I submit a new ticket?
+
+Depending on the priority of the ticket and your support SLA, one or many of the following might happen:
+
+* Very often VSHNeers will add comments to the ticket, asking for more information or requesting for your feedback. You will be notified by e-mail of these comments, and you can see the history of conversations in the Control Panel.
++
+.Comments in tickets
+image::ticket-comments.png[]
+
+* Your ticket will change its identification, from `SUPPORT` to the code of your support project. In that case, the layout of the ticket list will change significatly, showing the projects and its associated tickets inside, as shown in the picture below.
```

Comment:
This normally only happens when a ticket gets created via support@vshn.ch. I'm not sure why Inity GmbH behaves this way. Maybe @davidgubler can help and we might need to adapt this description here.

Reply:
Inity GmbH probably does not have a complete jira setup with its project and operations ticket, hence the control panel creates a SUPPORT ticket instead. This is not normal though and should not be used as a reference.

Reply:
Should I remove the paragraph above? What is the reference behaviour? Thanks.

Reply:
> Inity GmbH probably does not have a complete jira setup with its project and operations ticket

It has a completely regular Jira setup, like any other customer. It is meant for such things, demos, trials and education. Would you mind to check it?

> Should I remove the paragraph above? What is the reference behaviour? Thanks.

I would prefer to fix the behavior of Inity GmbH and then reword the sentences to match the actual situation then. And also update the screens then...

Reply:
I do not know how to proceed here, sorry. Maybe we could leave this paragraph out from the help pages for the moment and work on it later on.

Reply:
I changed the wording of this paragraph, please re-check to see if it fits a normal workflow.

Reply:
There is still something funky... I'm trying to figure out what and let you know.

Reply:
OK, I reread the docs, I would say it's fine now. Nevertheless, the demo customer Inity has some issues, tickets shouldn't be created by default in SUPPORT when there is a Jira project available. But that's another issue, not a documentation issue.